terminus-ui/link/src/link.component.ts

```diff
@@ -74,6 +83,12 @@ export class TsLinkComponent {
   @Input()
   public isExternal = false;
 
+  /**
+   * Decide whether an external icon should be shown
+   */
+  @Input()
```

Comment:
I don't believe we should expose this directly to consumers. They shouldn't have the option to remove the external icon for actual external links. I think what you've done using this flag internally is enough.